src/types/group.ts

```
@@ -0,0 +1,54 @@
+
+export interface Group {
+    admin: string,
+    created_at: string,
+    id: string,
+    metadata: string,
+    total_weight: string,
+    version: string
+}
+
+export interface GroupPolicy {
+}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)